fabric-convention-tags-v1/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/BlockTagGenerator.java

```diff
@@ -55,11 +54,76 @@ protected void configure(RegistryWrapper.WrapperLookup registries) {
 				.add(Blocks.BOOKSHELF);
 		generateGlassTags();
 		generateShulkerTag();
+		getOrCreateTagBuilder(ConventionalBlockTags.WOODEN_BARRELS)
+				.add(Blocks.BARREL);
+
+		generateBuddingTags();
+
+		getOrCreateTagBuilder(ConventionalBlockTags.VILLAGER_JOB_SITES)
+				.add(Blocks.BARREL)
+				.add(Blocks.BLAST_FURNACE)
+				.add(Blocks.BREWING_STAND)
+				.add(Blocks.CARTOGRAPHY_TABLE)
+				.add(Blocks.CAULDRON)
```

Comment:
Needs cauldron variants (though not the tag, as the game does not use it for this purpose)